components/mpas-ocean/src/Registry.xml

```diff
@@ -1078,6 +1078,10 @@
 					description="The density of land ice."
 					possible_values="Any positive real number"
 		/>
+		<nml_option name="config_land_ice_flux_rho_ocean" type="real" default_value="1028.0" units="kg m^-3"
+					description="ocean density used to calculate landIceDraft at floatation (assumed constant and uniform). Should be consistent with MALI's config_ocean_density. This is an alternative to the coupler variable effectiveDensityInLandIce which is not currently used."
+					possible_values="Any positive real number"
+		/>
```

Comment:
As discussed on Slack, let's maybe move this to a different (new?) namelist section and remove `flux` from the name.